src/renderer/components/Images/collaborate.tsx

```diff
@@ -0,0 +1,305 @@
+        <path
+            d='M75.9007 65.7776H113.2'
+            stroke='var(--center-channel-color)'
+            strokeOpacity='0.56'
+            strokeLinecap='round'
+        />
+        <path
+            d='M76.6321 22.6271H83.2144'
+            stroke='var(--center-channel-color)'
+            strokeOpacity='0.56'
+            strokeLinecap='round'
+        />
+        <path
+            d='M75.9009 55.0015H89.7968'
+            stroke='var(--center-channel-color)'
+            strokeOpacity='0.56'
+            strokeLinecap='round'
+        />
+        <path
+            d='M84.6769 22.6271H91.2592'
+            stroke='var(--center-channel-color)'
+            strokeOpacity='0.56'
+            strokeLinecap='round'
+        />
+        <path
+            d='M93.4536 55.0015H103.454'
+            stroke='var(--center-channel-color)'
+            strokeOpacity='0.56'
+            strokeLinecap='round'
+        />
+        <path
+            d='M106.454 55.0015H111.454'
+            stroke='var(--center-channel-color)'
+            strokeOpacity='0.56'
+            strokeLinecap='round'
+        />
+        <rect
+            x='91.2593'
+            y='29.9409'
+            width='21.9409'
+            height='11.7018'
+            rx='1'
+            fill='var(--center-channel-color)'
+            fillOpacity='0.32'
+        />
+        <g clipPath='url(#clip0_4216_119972)'>
+            <path
+                d='M81.7521 35.792L87.5055 36.8605C88.0891 33.6847 85.9851 30.6307 82.8085 30.0407C79.6304 29.4507 76.5808 31.5461 75.9987 34.7235C75.4151 37.8993 77.519 40.9534 80.6957 41.5433C81.656 41.7217 82.6046 41.6548 83.4778 41.3858L81.7521 35.792Z'
+                fill='var(--center-channel-bg)'
+            />
+            <path
+                d='M83.4778 41.3858L87.5055 36.8605C88.0891 33.6847 85.9851 30.6307 82.8085 30.0407C79.6304 29.4507 76.5823 31.5461 75.9987 34.7235C75.4151 37.8993 77.519 40.9534 80.6957 41.5433C81.656 41.7217 82.6046 41.6548 83.4778 41.3858Z'
+                fill='var(--center-channel-color)'
+                fillOpacity='0.32'
+            />
+            <path
+                d='M83.4773 41.3856C85.4912 40.7659 87.0987 39.0762 87.505 36.8603L81.7516 35.7918L83.4773 41.3856Z'
+                fill='var(--center-channel-color)'
+                fillOpacity='0.64'
+            />
+            <path
+                d='M83.6678 37.8343C84.7897 36.7699 84.8412 34.9922 83.7829 33.8638C82.7246 32.7354 80.9573 32.6836 79.8354 33.748C78.7136 34.8125 78.662 36.5902 79.7203 37.7186C80.7786 38.847 82.546 38.8988 83.6678 37.8343Z'
+                fill='var(--center-channel-bg)'
+            />
+        </g>
+        <path
+            d='M50.1064 105.143L62.1126 117V56C62.1126 54.8954 61.2171 54 60.1126 54H1.9999C0.895332 54 -9.91821e-05 54.8954 -9.91821e-05 56V102.566C-9.91821e-05 103.67 0.895332 104.566 1.9999 104.566H48.701C49.2272 104.566 49.7321 104.773 50.1064 105.143Z'
+            fill='var(--indigo-400)'
```

Comment:
I believe this color was included in the exported in the HTLM svgs but is not part of our defined variables. I replaced these with the constant value as they are not expected to be themed. 

Same with `--neutral` cases below.

Reply:
I just included the variables in our css_variables: https://github.com/mattermost/desktop/blob/5c593a58290ad0b60497b73d63f95633edad0736/src/renderer/css/_css_variables.scss#L3

Reply:
Didn't get to that part, you are one step ahead. Thanks!